doc-content/jbpm-docs/src/main/asciidoc/Persistence/transaction-cmt-proc.adoc

```diff
@@ -0,0 +1,45 @@
+[id='transaction-cmt-proc_{context}']
+= Configuring container-managed transactions
+
+If you embed the {PROCESS_ENGINE} in an application that executes in Container-managed transaction (CMT) mode, for example, EJB beans, you must complete additional configuration. This configuration is especially important if the application runs on an application server that does not allow a CMT application to access a `UserTransaction` instance from JNDI, for example, 
+WebSphere Application Server.
+
+The default transaction manager implementation in the {PROCESS_ENGINE} relies on `UserTransaction` to query transaction status and then uses the status to determine whether to start a transaction. In environments that prevent access to a `UserTransaction` instance, this implementation fails.
+
+To enable proper execution in CMT environments, the {PROCESS_ENGINE} provides a dedicated transaction manager implementation: 
+`org.jbpm.persistence.jta.ContainerManagedTransactionManager`. This transaction manager expects that the transaction is active and always returns `ACTIVE` when the `getStatus()` method is invoked. Operations such as `begin`, `commit`, and `rollback` are no-op methods, because the transaction manager can not affect these operations in container-managed transaction mode. 
+
+[NOTE]
+====
+During process execution your code must popagate any exceptions thrown by the engine to the container to ensure that the container rolls transactions back when necessary.
+====
+
+.Procedure
+
+To configure this transaction manager, complete the following steps:
```

Comment:
Need to move this line above the procedure label, as the last line of the intro:

```
To configure this transaction manager, complete the steps in this procedure.

.Procedure
. In your code ...
```